cpp/oneapi/dal/backend/primitives/reduction/test/reduction_rm_random_dpc.cpp

```diff
@@ -45,6 +46,13 @@ using reduction_types = std::tuple<std::tuple<float, sum<float>, identity<float>
                                    std::tuple<double, max<double>, identity<double>>,
                                    std::tuple<double, min<double>, identity<double>>>;
 
+using finiteness_types = std::tuple<std::tuple<float, sum<float>, identity<float>>,
```

Comment:
is it necessary to have sums?

Reply:
The finiteness checker for GPU will do it as a first step, I wanted to make sure that a sum with an infinite is checked. Let me know, I can drop that if you think its a good idea.